data/json/techniques.json

```diff
@@ -8,6 +8,7 @@
         "type" : "technique",
         "id" : "WBLOCK_1",
         "name" : "Block",
+        "//": "Block bonus of 4. Used by most basic weapons a competent combatant could put between themselves and an attack.",
```

Comment:
Instead of adding it as a comment, it should be a field:
```
block_ability: 4,
```
Also a field in the respective technique class.